examples/user_guide/WMTS_Example.ipynb

```diff
@@ -137,15 +122,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "meta"
+    "meta = quest.api.get_metadata(staged_id)\n",
+    "path = list(meta.values())[0]['file_path']\n",
+    "arr = xr.open_rasterio(path)\n",
+    "gv.tile_sources.Wikipedia * gv.RGB((arr.x, arr.y, arr[0].values, arr[1].values, arr[2].values), vdims=['R', 'G', 'B']).options(width=800, height=600)"
```

Comment:
@sdc50 This is the bit that loads the downloaded tile and displays it using geoviews.

Reply:
As @jlstevens suggested I'll likely add support for doing this automatically using ``gv.RGB.load_image``.

Reply:
@philippjfr Thanks!

Reply:
@sdc50 I just added ``gv.RGB.load_tiff`` to geoviews and am releasing v1.5.4a5. I'd recommend using that.